app/services/state_file/state_information_service.rb

```diff
@@ -46,13 +47,14 @@ def state_code_to_name_map
     STATES_INFO = IceNine.deep_freeze!({
       az: {
         intake_class: StateFileAzIntake,
-        navigation_class: Navigation::StateFileAzQuestionNavigation,
-        submission_builder_class: SubmissionBuilder::Ty2022::States::Az::IndividualReturn,
         mail_voucher_address: "Arizona Department of Revenue<br/>" \
                               "PO Box 29085<br/>" \
                               "Phoenix, AZ 85038-9085".html_safe,
+        navigation_class: Navigation::StateFileAzQuestionNavigation,
         primary_tax_form_name: "Form AZ-140V",
         state_name: "Arizona",
+        schema_file_path: ["us_states", "unpacked", "AZIndividual2023v1.0", "AZIndividual", "IndividualReturnAZ140.xsd"],
```

Comment:
Agreed that duplicating this path could lead to a bug if we ever changed the path. Alternatively to documenting it, we could make `schema_file_loader.rb` provide a public method that returns the path, so that the information is only in one place. I think the benefit is small, so I don't have a strong opinion about which approach should be taken.

Reply:
ohh maybe i'll try out the public method idea